NDTensors/src/backup/arraystorage/blocksparsearray/storage/unwrap.jl

```diff
@@ -1 +1 @@
-Unwrap.parenttype(::Type{<:BlockSparseArray{<:Any,<:Any,P}}) where {P} = P
+TypeParameterAccessors.parenttype_position(::Type{<:BlockSparseArray}) = 3
```

Comment:
I forget the conclusion, but I believe we may have to go back to returning `Position(3)` here (and additionally use the new `parameter(::Type{<:BlockSparseArray}, parenttype)` syntax).

Reply:
For right now I am using `Position` to make sure that the code is type stable. If in the future it works with `Int` we can revert back to that.